### PR TITLE
remove linebreaks in SpecName macro arguments

### PR DESCRIPTION
**Pull request by peterbe**
> What was wrong/why is this fix needed? (quick summary only)

There are lots of cases of calling the `SpecName()` KS macro with a string that contains a line break. 
This is, the beginning, of a solution to automate cleaning this up.

### CODE REVIEW

**Comment by peterbe** [comment]
@sideshowbarker I updated my Python script. Let's see if this works better. Then I'll do another 50 files or so.

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
> @sideshowbarker I updated my Python script. Let's see if this works better.

Yup — looks good to me

**Comment by peterbe** [comment]
@sideshowbarker Pushed another 50 files. r?

**Comment by peterbe** [comment]
I kinda wish it was:
```
{{SpecName('Media Session', '#dom-mediapositionstate-position', 'MediaPositionState.position')}}
```
instead of
```
{{SpecName('Media Session','#dom-mediapositionstate-position','MediaPositionState.position')}}
```
(i.e. a trailing space after each `,`)
But perhaps another day.

**Review by sideshowbarker — APPROVED** [review]
LGTM

**Comment by peterbe** [comment]
Let's just do all of them! So final r?

**Comment by peterbe** [comment]
Also, feel free to merge when/if you think it's good.

File: files/en-us/web/api/element/removeattribute/index.html [hunk 1]
```diff
@@ -62,8 +62,7 @@ <h2 id="Specifications">Specifications</h2>
   </thead>
   <tbody>
     <tr>
-      <td>{{SpecName('DOM WHATWG', '#dom-element-removeattribute', 'Element"
-        removeAttribute')}}</td>
+      <td>{{SpecName('DOM WHATWG', '#dom-element-removeattribute', 'Element" removeAttribute')}}</td>
```
Comment (sideshowbarker) [inline files/en-us/web/api/element/removeattribute/index.html:65]:
```suggestion
      <td>{{SpecName('DOM WHATWG', '#dom-element-removeattribute', 'Element: removeAttribute')}}</td>
```
typo that we may as well fix while we’re in here…

**Review by sideshowbarker — APPROVED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**